src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
I don't think we'll need to. There's plenty of fine grained control already now.